requirements/env_common.yml

```diff
@@ -22,7 +22,8 @@ dependencies:
     - pyGPGO
     - pymatgen
     - simdna
-    - transformers==4.10.*
+    - transformers
+    - tokenizers
```

Comment:
If `tokenizers` is a new soft requirement, we should add it to the soft requirements table in the docs.